src/test/java/network/brightspots/rcv/SecurityTests.java

```diff
@@ -156,4 +156,16 @@ void testFailureOnInvalidOutputDirectory() {
         },
         errors.toArray());
   }
+
+  @Test
+  @DisplayName("Ensure FIPS Compliance check is run")
+  void testFipsComplianceWasCorrectlySetUp() {
+    Assertions.assertTrue(SecurityConfig.haveProvidersBeenCulled());
```

Comment:
So I guess this is something that gets set up (i.e. all of `SecurityConfig` is run) when the app initializes, but it only _matters_ / gets checked during Hart tabulation?

Reply:
Yep, though I'd maybe use the term `verified` over `matters`, in that it may _affect_ other places in the code (and some may take "matters" to mean "affects")